sympy/parsing/latex/latex_parser.py

```diff
@@ -0,0 +1,128 @@
+import operator
```

Comment:
Is this file also automatically generated?

Reply:
I guess it probably isn't :)

Reply:
No this file is not auto generated